script/Ciel/Day 14/14_01_CIEL14_1.txt

```diff
@@ -235,7 +237,9 @@ However, I did come to know hatred for the first time.
 -- この失望もこの不純も、全て、あの女さえいなければ起きえない<感|も><情|の>だったのだから。
 // Uhh real 'she was dressed provocatively your honour' mindset.
 // Sorry to say that Roa is cancelled.
-Because if she hadn't existed, then these %{i}things%{/i}――my despair, my impurity, none of it――could never have come to be.
+Because if she hadn't existed, then these %{i}things%{/i}――
+my despair, my impurity, none of it――
+could ever have come to be.
```

Comment:
I don't think this sentence works. Would move the 'none' up.

> Because if she hadn't existed, then none of these %{i}things%{/i}――
> my despair, my impurity――
> could ever have come to be.